config/env/all.js

```diff
@@ -3,7 +3,7 @@ const port = process.env.PORT || 4000;
 let db = process.env.MONGOLAB_URI || process.env.MONGODB_URI;
 
 if (!db) {
-    db = process.env.NODE_ENV === 'test' ? 'mongodb://localhost:27017/nodegoat' : 'mongodb://nodegoat:owasp@nodegoat-cluster-shard-00-00.zkkl5.mongodb.net:27017,nodegoat-cluster-shard-00-01.zkkl5.mongodb.net:27017,nodegoat-cluster-shard-00-02.zkkl5.mongodb.net:27017/nodegoat?ssl=true&replicaSet=atlas-qajmlr-shard-0&authSource=admin&retryWrites=true&w=majority';
+    db = 'mongodb://localhost:27017/nodegoat';
 }
```

Comment:
```suggestion
let db = process.env.MONGODB_URI || "mongodb://localhost:27017/nodegoat";
```
This would revert back to how it was in c44bbd6.

MONGOLAB_URI isn't needed any more, since mLab got bought out and is moving everyone over to MongoDB Atlas. Might as well make the DB setup code match the port setup line above.